examples/bri-3/prisma/schema.prisma

```diff
@@ -28,3 +28,15 @@ model Workstep {
   securityPolicy String
   privacyPolicy  String
 }
+
+model Transaction {
+  transactionId String @id
```

Comment:
id instead of transactionId to be consistent

Reply:
+1. I ll change it.